build.gradle.kts

```diff
@@ -88,6 +88,28 @@ allprojects {
       useJUnitPlatform {}
       maxParallelForks = Runtime.getRuntime().availableProcessors()
     }
+
+    if (project.hasProperty("alsoTestAgainstJava8")) {
+      afterEvaluate {
+        val javaToolchains = extensions.findByType(JavaToolchainService::class.java)
+        if (javaToolchains != null) {
+          val testWithJava8 =
+            tasks.register<Test>("testWithJava8") {
```

Comment:
doesnt this mean that gradle execution in CI should mention a `testWithJava8` task being executed?
i am not seeing it in the CI run on this PR :thinking:

Reply:
Yea - already fixed that